src/components/CreateEnvironment/index.tsx

```diff
@@ -218,14 +220,13 @@ export default function CreateEnvironment() {
               <Button
                 variant="contained"
                 startIcon={<AddIcon />}
-                disabled={
-                  envBuildInFlight ||
-                  name.length === 0 ||
-                  description.length === 0 ||
-                  path.length === 0 ||
-                  (path !== "users/" + username && !groups.includes(path.split("/")[1])) ||
-                  validPackages.length === 0
-                }
+	        data-reason={disabled ? [
```

Comment:
envBuildInFlight is a reason to be disabled, but not provided as a reason in the popup?

Reply:
Have added a message for when `envBuildInFlight` is set.